sw/airborne/firmwares/rotorcraft/guidance/guidance_h.c

```diff
@@ -555,6 +555,7 @@ void guidance_h_nav_enter(void)
   reset_guidance_reference_from_current_position();
 
   nav_heading = stateGetNedToBodyEulers_i()->psi;
+  guidance_h.sp.heading = stateGetNedToBodyEulers_f()->psi;
```

Comment:
I think it is fine (although I haven't tested it), maybe just mention it in the PR description...